centaur/src/it/scala/centaur/AbstractCentaurTestCaseSpec.scala

```diff
@@ -44,14 +45,17 @@ abstract class AbstractCentaurTestCaseSpec(cromwellBackends: List[String], cromw
   def executeStandardTest(testCase: CentaurTestCase): Unit = {
     def nameTest = s"${testCase.testFormat.testSpecString} ${testCase.workflow.testName}"
 
-    def runTest(): IO[SubmitResponse] = testCase.testFunction.run
+    def runTestAndDeleteZippedImports(): IO[SubmitResponse] = for {
+      submitResponse <- testCase.testFunction.run
+      _ = cleanUpImports(testCase.workflow.data) // cleanup imports in the end of test
```

Comment:
In a for-comp, this second line will only run if the first line succeeds. Presumably even if the test fails, we still want to delete this zip file?

Reply:
We want to avoid this file being deleted between intermittent test failures. Otherwise retry attempt will fail, on attempt to submit workflow with missing imports archive.
And in case of the final failure I think we may not care about the amount of free space left - build will already fail anyway.

Reply:
Good point!